src/test/scala/rxscalajs/ObservableTest.scala

```diff
@@ -7,11 +7,12 @@ import utest._
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.scalajs.js
+import scala.scalajs.js.|
 
 
 object ObservableTest extends TestSuite {
 
-
+  type Creator = Unit | (() => Unit)
```

Comment:
I think type Creator needs to explicitly be `js.Function0[Unit]`, for JS interop to work correctly